manifest.yaml

```diff
@@ -46,6 +46,7 @@ box:
         darwin-arm64: livepeer-mistserver-darwin-arm64.tar.gz
         linux-amd64: livepeer-mistserver-linux-amd64.tar.gz
         linux-arm64: livepeer-mistserver-linux-arm64.tar.gz
+        #TODO: automate mist builds
```

Comment:
What is this comment? I think it's not clear enough, so I'd rather recommend removing it (and create a GH Issue instead).